simphony/testing/tests/test_utils.py

```diff
@@ -356,6 +841,18 @@ def test_create_particles(self):
                 particle.data, create_data_container(constant=index),
                 testcase=self)
 
+    def test_create_particles_with_id(self):
```

Comment:
Oh, I see, sorry I was not aware of that PR. We can remove this as soon as #187 is merged